schemas/context/environment.schema.json

```diff
@@ -38,11 +38,29 @@
           "description":
             "The name of the operating system used when the observation was made. This attribute should not contain any version information i.e. 10.5.3, but can contain *edition* designations such as 'Ultimate', or 'Professional'."
         },
+        "xdm:operatingSystemMinor": {
```

Comment:
This separation of system and system minor is clearly arbitrary and not based on any known standard.

Reply:
Removed minor